qiskit/providers/__init__.py

```diff
@@ -500,6 +546,110 @@ def result(self):
         def status(self):
             return JobStatus.DONE
 
+======================================
+Migrating between Backend API Versions
+======================================
+
+BackendV1 -> BackendV2
+======================
+
+The :obj:`~BackendV2` class re-defined user access for most properties of a
+backend to make them work with native Qiskit data structures and have flatter
+access patterns. However this means when using a provider that upgrades
+from :obj:`~BackendV1` to :obj:`~BackendV2` existing access patterns will need
+to be adjusted. It is expected for existing providers to deprecate the old
+access where possible to provide a graceful migration, but eventually users
+will need to adjust code. The biggest change to adapt to in :obj:`~BackendV2` is
+that most of the information accesible about a backend is contained in it's
+:class:`~qiskit.transpiler.Target` object and the backend's attributes often query
+it's :attr:`~qiskit.providers.BackendV2.target`
+attribute to return information, however in many cases the attributes only provide
+a subset of information the target can contain. For example, ``backend.coupling_map``
+returns a :class:`~qiskit.transpiler.CouplingMap` constructed from the
+:class:`~qiskit.transpiler.Target` accesible in the
+:attr:`~qiskit.providers.BackendV2.target` attribute, however the target may contain
+instructions that operate on more than two qubits (which can't be represented in a
+:class:`~qiskit.transpiler.CouplingMap`) or has instructions that only operate on
+a subset of qubits (or two qubit links for a two qubit instruction) which won't be
+detailed in the full coupling map returned by
+:attr:`~qiskit.providers.backend_v2.coupling_map`. So depending on your use case
```

Comment:
This crossref looks funky - it's not linked, and its role is `attr` but there's no class in it.

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)